<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demo</class>
 <widget class="QMainWindow" name="demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>710</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QDockWidget" name="dockWidget_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>524287</height>
       </size>
      </property>
      <property name="windowTitle">
       <string>工具箱</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2">
       <widget class="QPushButton" name="show">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>显示字符</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clear">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>清除内容</string>
        </property>
       </widget>
       <widget class="QPushButton" name="font">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}
</string>
        </property>
        <property name="text">
         <string>设置字体</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ascNum">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>输入ASCII码</string>
        </property>
       </widget>
       <widget class="QPushButton" name="grayBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>灰度</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>480</y>
          <width>150</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="singleStep">
         <number>3</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>510</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>亮度调节</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>530</y>
          <width>150</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>-50</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="singleStep">
         <number>3</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>37</x>
          <y>560</y>
          <width>90</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>饱和度调节</string>
        </property>
       </widget>
       <widget class="QPushButton" name="blurBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>均值滤波</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sobelBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>125</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>Sobel边缘检测</string>
        </property>
       </widget>
       <widget class="QPushButton" name="prewittBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>125</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>Prewitt边缘检测</string>
        </property>
       </widget>
       <widget class="QPushButton" name="robertsBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>125</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>Roberts边缘检测</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addOrgBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>440</y>
          <width>125</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover
{
    background:rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>原图复合</string>
        </property>
       </widget>
       <widget class="QPushButton" name="mirrorHorBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image:url(:/img/images/mirror.svg)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="mirrorVerBtn">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image:url(:/img/images/mirror_2.svg)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="blackwhiteBtn">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image:url(:/img/images/blackwhite.svg)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="rotateleftBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>45</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;旋转&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image:url(:/img/images/rotate_right.svg)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="rotaterigBtn">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>45</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;旋转&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image:url(:/img/images/rotate_left.svg)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{border:5px solid rgb(255, 255, 255);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="restoreBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border-image:url(:/img/images/restore.svg)
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border-image:url(:/img/images/save.svg)
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveAsBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border-image:url(:/img/images/saveas.svg)
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_Dock_D"/>
    <addaction name="action_V"/>
    <addaction name="action_V_2"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>语言(&amp;L)</string>
    </property>
    <addaction name="action_L"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_S"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Dock_D">
   <property name="text">
    <string>显示Dock(&amp;D)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于本软件</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>打开图片(&amp;P)</string>
   </property>
  </action>
  <action name="action_V_2">
   <property name="text">
    <string>打开视频(&amp;V)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
